cfgov/jinja2/v1/_layouts/content-base.html

```diff
@@ -1,7 +1,11 @@
 {% extends 'base.html' %}
 
 {% block content scoped %}
-    <main class="content {% block content_modifiers -%}{%- endblock %}" id="main" role="main">
+    {%- import 'macros/accessible-languages.html' as accessible_languages with context-%}
+    <main class="content {% block content_modifiers -%}{%- endblock %}"
+          id="main"
+          role="main"
+          {{ accessible_languages.render() }}>
```

Comment:
Trivial: If you throw a `-` on the opening output tag (`{{-`), the `>` will slide up next to the `role` attribute if `accessible_languages.render()` results in no output, which could possibly happen if this template is used for something that has no `page` object. (Like my upcoming non-Wagtail wellbeing pages.)